### PR TITLE
UIView: -(CGPoint)sam_trueCenter

### PR DESCRIPTION
**Pull request by troyharris**
Returns the true center of the view, regardless of orientation

PS: I went through these and some are really useful. I might slowly integrate my own utility categories into my fork and I'll pull request any I think might be useful. Since I don't really know what you use this for, feel free to deny. ;)

### CODE REVIEW

**Comment by soffes** [comment]
Can you help me understand why you would use this?

**Comment by troyharris** [comment]
Because of this: http://stackoverflow.com/questions/8180857/landscape-mode-uiview-center-gives-potrait-mode-coordinates

Obviously you could use bounds as the answer states (and divide width and height by 2). Same result just different way to skin the cat. Nothing too exciting but I seem to use it in all of my projects.

File: SAMCategories/UIKit/UIView+SAMAdditions.h [hunk 1]
```diff
@@ -91,4 +91,10 @@
  */
 - (id)sam_firstSuperviewOfClass:(Class)superviewClass;
 
+/**
+ Returns the true center of the view, regardless of orientation
+ */
+
```
Comment (soffes) [inline SAMCategories/UIKit/UIView+SAMAdditions.h]: :scissors: no new line here

File: SAMCategories/UIKit/UIView+SAMAdditions.m [hunk 2]
```diff
@@ -83,4 +83,10 @@ - (id)sam_firstSuperviewOfClass:(Class)superviewClass {
 	return nil;
 }
 
+- (CGPoint)sam_trueCenter {
+    UIInterfaceOrientation orientation = [UIApplication sharedApplication].statusBarOrientation;
+    CGPoint trueCenter = UIInterfaceOrientationIsLandscape(orientation) ? CGPointMake(self.center.y, self.center.x) : self.center;
```
Comment (soffes) [inline SAMCategories/UIKit/UIView+SAMAdditions.m]: Let's just return it on 88 instead of making a variable and immediately returning it.

**Comment by soffes** [comment]
To minor nitpicks and then I'll merge it :)

**Comment by troyharris** [comment]
There you go! Thanks again for this repo. I'll definitely be including it in future projects.

**Comment by soffes** [comment]
Thanks!

**Pull request merged**